docs/documentation/admin/admin_accessrole.md

```diff
@@ -71,63 +66,85 @@ The following table contains the default _access roles_ that already exist in an
 
 ### Default Permissions {#default-permissions}
 Default _permissions_ come pre-installed and can be assigned to any _access role_.
+The following table contains the default _permissions_ present in all company configurations.
 
-_Users_ need either _admin-accesscontrol-read permission_ or _admin-accesscontrol-write permission_ to read or write permissions respectively.
+:::note
+_Admin-accesscontrol-read permission_ or _admin-accesscontrol-write permission_ are required to read or write permissions respectively.
+:::
 
-The next table contains the default _permissions_ present in all company configurations.
+| Permissions for Admin Access | Description |
```

Comment:
En general, al ver la imagen del arbol, todos los permisos que tienen un color, permiten ver la seccion a la que pertenecen)

Por ejemplo:

admin-*-read: Permite ver la seccion de Administrador. Permite ver todos los elementos.
admin-*-write: Permite ver la seccion de Administrador. Permite ver y editar cualquier elemento.
admin-access: Permite ver la seccion de Administrador. (no estoy bien segura que exactamente deberia ver, creo que deberia poder entrar a la seccion, pero no ver elementos *idealmente consultar con el Eddie )

Reply:
@acedward Dos preguntas:

1. Aquí no sé como traducir exactamente **elementos** en, p.e., "Premite ver (editar) todos los _elementos_". 
    - Mirando el árbol, diría: "Allows viewing (reading) all _endpoints_". 
    - ¿Está bien usar _endpoints_ o hay algún término preferido?

2. Para **Admin-access**, escribí: 
    - "Grants access to the Administrator section, but will only show the _endpoint permissions_ the user has been given." 
    - ¿Está bien esa descripción? (Y nuevamente, el término "endpoint" está bien utilizado?)